### PR TITLE
Feature/user navigation

### PR DESCRIPTION
**Pull request by CesarCachay**
This pull request allow us:
- Open portal inside modal
- Fix the navigation of the user

### CODE REVIEW

File: simcase-game/src/App.js [hunk 1]
```diff
@@ -1,22 +1,49 @@
...
+        setIsModalOpen={setIsModalOpen}
+        openModal={openModal}
+        closeModal={closeModal}
       />
-      <Header />
-      <Welcome />
-      <Role />
-      <Walkthrough />
-    </>
+      <GameUI
```
Comment (dotconde) [inline simcase-game/src/App.js]: Well done, this is what we were looking for inside GameUI. Thanks Cesar

File: simcase-game/package.json [hunk 2]
```diff
@@ -12,7 +12,8 @@
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
     "react-route": "^1.0.3",
-    "react-scripts": "3.0.1"
+    "react-scripts": "3.0.1",
```
Comment (dotconde) [inline simcase-game/package.json]: Why did you delete previously this dependency ?

**Review by dotconde — APPROVED** [review]

**Comment by dotconde** [comment]
Branch needs to be solved

File: simcase-game/src/App.js [hunk 3]
```diff
@@ -1,22 +1,49 @@
 /** @jsx jsx */
 import React from "react";
-import { jsx, Global } from "@emotion/core";
-import Welcome from "./components/Welcome";
-import Role from "./components/Role";
-import Walkthrough from "./components/Walkthrough";
-import Header from "./components/Header";
+import { jsx } from "@emotion/core";
```
Comment (test0n3) [inline simcase-game/src/App.js]: Needs to reference Global so <Global /> can work.

**Review by test0n3 — COMMENTED** [review]
Mandatory to solve conflicts.

**Review by test0n3 — APPROVED** [review]
